website/docs/r/search_deployment.html.markdown

```diff
@@ -1,57 +1,99 @@
 ---
 layout: "mongodbatlas"
-page_title: "MongoDB Atlas: search deployment"
-sidebar_current: "docs-mongodbatlas-resource-search-deployment"
+page_title: "MongoDB Atlas: mongodbatlas_search_deployment"
+subcategory: "docs_mongodbatlas_search_deployment_resource"
 description: |-
-    Provides a Search Deployment resource.
+    "Provides a Search Deployment resource."
 ---
 
 # Resource: mongodbatlas_search_deployment
 
-`mongodbatlas_search_deployment` provides a Search Deployment resource. The resource lets you create, edit and delete dedicated search nodes in a cluster.
 
--> **NOTE:** For details on supported cloud providers and existing limitations you can visit the [Search Node Documentation](https://www.mongodb.com/docs/atlas/cluster-config/multi-cloud-distribution/#search-nodes-for-workload-isolation).
 
--> **NOTE:** Only a single search deployment resource can be defined for each cluster.
```

Comment:
Would prefer to not remove this content as it is valuable for the docs. Could we add this in the template, or as part of the description field of the schema?

Reply:
Could you clarify the content that you want to include in the template? is the NOTE?

Reply:
Yes the 2 `NOTE:` provide valuable info/clarification.

Reply:
yes, we should add them in the template in this case